Automattic/jetpack
# Decode non-XML entities for translations

Fixes #3857

#### Changes proposed in this Pull Request:
This replaces named HTML entities, for example `&rsquo;` as it is used in French translations, so that it doesn't break the XSLT.

To test:
 - set the language to French
 - delete the `jetpack_sitemap_xsl` transient
 - access `/sitemap.xsl` and check that no named entity appears there
